### PR TITLE
net/coova-chilli: Coova-Chilli V1.3.1.4

### PR DESCRIPTION
**Pull request by Spomky**
Maintainer: Imre Kaloz <kaloz@openwrt.org>
Compile tested: OpenWRT 15.05.1 on ar71xx
Run tested: OpenWRT 15.05.1 on ar71xx, installation is correct

Description:

This commit includes modifications to allow the installation of Coova-Chilli v1.3.1.4

Signed-off-by: Florent Morselli <florent+git@morselli.fr>

### CODE REVIEW

**Comment by hnyman** [comment]
> Maintainer: ?

From Makefile:  PKG_MAINTAINER:=Imre Kaloz <kaloz@openwrt.org>

CC to maintainer: @kaloz

**Comment by Spomky** [comment]
Done

**Comment by diizzyy** [comment]
Nice, change your commit msg to net/coova-chili: Update to version 1.3.14
Preferably don't touch the OpenWRT copyright.

File: net/coova-chilli/Makefile [hunk 1]
```diff
@@ -8,32 +8,98 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=coova-chilli
-PKG_VERSION:=1.3.0+20141128
+PKG_VERSION:=1.3.1.4+20160523
```
Comment (diizzyy) [inline net/coova-chilli/Makefile]: Can you bump this to 1.4?

File: net/coova-chilli/Makefile [hunk 2]
```diff
@@ -8,32 +8,98 @@
...
 PKG_MAINTAINER:=Imre Kaloz <kaloz@openwrt.org>
 PKG_LICENSE:=GPL-2.0+
 PKG_LICENSE_FILES:=COPYING
-PKG_RELEASE:=4
+PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=git://github.com/coova/coova-chilli
```
Comment (diizzyy) [inline net/coova-chilli/Makefile]: Please change this top https

**Comment by diizzyy** [comment]
@Spomky 
You also need to squash your commits

**Comment by Spomky** [comment]
Yes I will do it. 
Commits con also be squashed when a PR is merged.

File: net/coova-chilli/Makefile [hunk 3]
```diff
@@ -8,32 +8,98 @@
...
-PKG_RELEASE:=4
+PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_URL:=git://github.com/coova/coova-chilli
+PKG_SOURCE_URL:=https://github.com/coova/coova-chilli
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
-PKG_SOURCE_VERSION:=b93de20a288c01c2ba28e96e31ad6da01627f45f
+PKG_SOURCE_VERSION:=428a0504e1e0153a969d4f9209aba4f3d4df98a3
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
```
Comment (diizzyy) [inline net/coova-chilli/Makefile]: Preferably use xz instead gz

**Comment by diizzyy** [comment]
@Spomky 
Can you fix the remaining issues so we won't have this pull request open until the end of time? :-)

**Comment by diizzyy** [comment]
@Spomky 
If you want this in LEDE release, please update your pull request.
Also, squash your commits.

**Pull request closed without merging**